common/primitives/src/node.rs

```diff
@@ -36,7 +36,7 @@ pub type Signature = MultiSignature;
 /// Index of a transaction in the chain.
 pub type Index = u32;
 
-/// the time period in blocks that Staking Rewards are based upon
+/// the type for the index of the Staking Reward time period
```

Comment:
is this clearer or muddier?

Reply:
imho, muddier

Reply:
```suggestion
/// the index of the specific interval in the Staking Reward time period
```

Reply:
I think it's fine to say that it is "the type used for the index of the Staking Reward time period", since this type is used to specify that `reward_era`(the index) is a `u32`